features/page_objects/back_office/new/registration_edit_completed_page.rb

```diff
@@ -0,0 +1,5 @@
+class RegistrationEditCompletedPage < SitePrism::Page
```

Comment:
If it was me I'd rename these new files without `registration_` just because it's less typing... I prefer `edit_completed_page` - but that's just my preference.

Reply:
I can do that :)